bentoml/service_env.py

```diff
@@ -141,7 +139,7 @@ def __init__(
         self._python_version = PYTHON_VERSION
 
         self._conda_env = CondaEnv(
-            "bentoml-" + bento_service_name,
+            CONDA_ENV_DEFAULT_NAME,
```

Comment:
that's a great call! the conda env name was not used anyway. thank you for fixing this!